src/Command/SyncDefinitionsCommand.php

```diff
@@ -47,6 +49,12 @@ protected function execute (InputInterface $input, OutputInterface $output) : in
 
 		$spaceInfo = $this->contentApi->getSpaceInfo();
 
+		if (!$this->environment->isProduction())
+		{
+			$io->caution("Reject to sync structure to Storyblok in non-production environment.");
+			return self::SUCCESS;
+		}
```

Comment:
This would also fail for development environments, unless we set the `HOSTING_TIER` (for the command execution) to `production`.

Is this something that we want, or should we only check against staging environment?

Reply:
That is intended. If you really want to sync, you can always set the env var for this call.

But we must make sure not to accidentally wipe the production instance.

Reply:
Should we add a reminder about `HOSTING_TIER="production" …` to this caution message to improve DX?

Reply:
No, I would make it intentionally "hard" to circumvent it.

Reply:
We now allow to overwrite it interactively.